src/vs/workbench/contrib/terminal/browser/terminal.ts

```diff
@@ -916,6 +916,8 @@ export interface ITerminalInstance {
 	registerQuickFixProvider(...options: ITerminalQuickFixOptions[]): void;
 
 	freePortKillProcess(port: string): Promise<void>;
+
+	runCommand(command: string, alt?: boolean): void;
```

Comment:
Let's move this to just after sendText/sendPath as it's closely related, also adding docs and renaming alt to something more descriptive like `addNewLine?: boolean` or `executeCommand?: boolean` would be good.